src/core/ext/filters/client_channel/http_proxy.cc

```diff
@@ -57,23 +57,24 @@
 namespace grpc_core {
 namespace {
 
-bool ServerInCIDRRange(std::string server_host,
+bool ServerInCIDRRange(absl::string_view server_host,
                        absl::string_view no_proxy_entry) {
   auto server_address = StringToSockaddr(server_host, 0);
   if (!server_address.ok()) {
     return false;
   }
-  std::vector<absl::string_view> cidr =
-      absl::StrSplit(no_proxy_entry, '/', absl::SkipEmpty());
-  if (cidr.size() != 2) {
+
+  std::pair<absl::string_view, absl::string_view> possible_cidr =
+    absl::StrSplit(no_proxy_entry, absl::MaxSplits('/', 2));
```

Comment:
Don't you still want `absl::SkipEmpty()` here?  Note that you can use both that and `MaxSplits()`:

```
absl::StrSplit(no_proxy_entry, absl::MaxSplits('/', 2), absl::SkipEmpty());
```